lineman/app/components/group_page/cover_photo_form/cover_photo_form.haml

```diff
@@ -0,0 +1,21 @@
+.group-cover-photo-modal
+  .lmo-disabled-form{ng-show: 'isDisabled'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
```

Comment:
aria-hidden: 'true' (note the single quotes) or it won't work.
Yeah.. we're overdue for a directive for this.